commit 5bf161a64f885b2b8780bbe1a5b20e892b0ea30f
Author: Richard Imaoka <richard.e.imaoka@gmail.com>
Date:   Fri May 27 00:25:10 2022 +0900

    tranform: translate(-648px)

diff --git a/client/app/src/App.tsx b/client/app/src/App.tsx
index f00c40b..2902cce 100644
--- a/client/app/src/App.tsx
+++ b/client/app/src/App.tsx
@@ -33,6 +33,8 @@ const CarouselContainer = ({ imagePathList }: CarouselContainerProps) => {
         style={{
           display: "flex",
           flexDirection: "row",
+          transition: "all 0.3s linear",
+          transform: `translate(-648px)`,
         }}
       >
         {imagePathList.map((path) => (
