commit cc54e35b736a88662544d2d3fe1fb51ab5b8e89b
Author: Richard Imaoka <richard.e.imaoka@gmail.com>
Date:   Fri May 27 00:35:25 2022 +0900

    centering buttons

diff --git a/client/app/src/App.tsx b/client/app/src/App.tsx
index b0efb86..8e8c77d 100644
--- a/client/app/src/App.tsx
+++ b/client/app/src/App.tsx
@@ -29,7 +29,6 @@ const CarouselContainer = ({
   return (
     <div
       style={{
-        width: "648px",
         overflowX: "hidden",
       }}
     >
@@ -54,11 +53,25 @@ const CarouselControl = () => {
   const [snapped, setSnapped] = useState(0);
 
   return (
-    <div>
+    <div
+      style={{
+        width: "648px",
+      }}
+    >
       <CarouselContainer imagePathList={imagePathList} snappedIndex={snapped} />
-      <button onClick={() => setSnapped(0)}>1</button>
-      <button onClick={() => setSnapped(1)}>2</button>
-      <button onClick={() => setSnapped(2)}>3</button>
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "center",
+          padding: "4px 0px",
+        }}
+      >
+        {imagePathList.map((path, index) => (
+          <button key={path} onClick={() => setSnapped(index)}>
+            {index + 1}
+          </button>
+        ))}
+      </div>
     </div>
   );
 };
