commit e2174d824238a3284a32f90c25f4bc89852629c3
Author: Richard Imaoka <richard.e.imaoka@gmail.com>
Date:   Fri May 27 00:29:10 2022 +0900

    snap state

diff --git a/app/src/App.tsx b/app/src/App.tsx
index b17822c..6e8dab6 100644
--- a/app/src/App.tsx
+++ b/app/src/App.tsx
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
 
 interface CarouselItemProps {
   imagePath: string;
@@ -47,12 +47,15 @@ const CarouselContainer = ({ imagePathList }: CarouselContainerProps) => {
 
 const CarouselControl = () => {
   const imagePathList = ["/images/1.png", "/images/2.png", "/images/3.png"];
+  const [snapped, setSnapped] = useState(0);
+
   return (
     <div>
       <CarouselContainer imagePathList={imagePathList} />
-      <button>1</button>
-      <button>2</button>
-      <button>3</button>
+      <button onClick={() => setSnapped(0)}>1</button>
+      <button onClick={() => setSnapped(1)}>2</button>
+      <button onClick={() => setSnapped(2)}>3</button>
+      <span>snapped ={snapped + 1}</span>
     </div>
   );
 };
